### PR TITLE
Batch disable users

### PR DESCRIPTION
**Pull request by alexanderzobnin**
Functions that can disable users and revoke tokens in a batch.
This is a part of #11188 and https://github.com/grafana/grafana-enterprise/issues/74
Related to https://github.com/grafana/grafana-enterprise/issues/78, https://github.com/grafana/grafana-enterprise/issues/77

### CODE REVIEW

File: pkg/services/sqlstore/user_test.go [hunk 1]
```diff
@@ -175,6 +175,30 @@ func TestUserDataAccess(t *testing.T) {
...
+
+				err = BatchDisableUsers(&disableCmd)
+				So(err, ShouldBeNil)
+
+				Convey("Should disable all provided users", func() {
+					query := m.SearchUsersQuery{}
+					err = SearchUsers(&query)
+
+					So(query.Result.TotalCount, ShouldEqual, 5)
+					So(query.Result.Users[0].IsDisabled, ShouldBeTrue)
```
Comment (xlson) [inline pkg/services/sqlstore/user_test.go]: This test depends on the returned order of users from the database unless I'm mistaken. From what I could tell we don't have any ordering setup when doing the query, so it seems like a bad idea to depend on the order (and is probably the reason why the Postgres integration tests are failing).

**Review by xlson — APPROVED** [review]
Looks great!

Can merge after fixing the failing test.

**Comment by alexanderzobnin** [comment]
@xlson tests fixed, so could you take a look, is it clear enough?

File: pkg/services/auth/auth_token.go [hunk 2]
```diff
@@ -305,6 +306,36 @@ func (s *UserAuthTokenService) RevokeAllUserTokens(ctx context.Context, userId i
 	})
 }
 
+func (s *UserAuthTokenService) BatchRevokeAllUserTokens(ctx context.Context, userIds []int64) error {
```
Comment (markelog) [inline pkg/services/auth/auth_token.go]: Shouldn't we add an event listener, so we could use `Dispatch`? Since right now we are bound to it?

Comment (xlson) [reply pkg/services/auth/auth_token.go]: UserAuthTokenService is used as a service and is injected so for this func it should not be necessary.

File: pkg/services/sqlstore/user.go [hunk 3]
```diff
@@ -487,6 +487,31 @@ func DisableUser(cmd *m.DisableUserCommand) error {
 	return err
 }
 
+func BatchDisableUsers(cmd *m.BatchDisableUsersCommand) error {
```
Comment (xlson) [inline pkg/services/sqlstore/user.go]: This one should be added to the bus though (thanks @markelog for making me think of it).

**Review by xlson — APPROVED** [review]
LGTM!

**Pull request merged**